Website/webb/src/pages/HomePage.js

```diff
@@ -29,86 +34,134 @@ const HomePage = () => {
   const [displayingRoute, setDisplayingRoute] = useState(false);
   const [foundRoute, setFoundRoute] = useState(false);
 
+  const [toggle, setToggle] = useState(false);
```

Comment:
nit: this hook's name should be updated to be more descriptive of what it's toggling